experimental/GModule/Cohomology.jl

```diff
@@ -2055,8 +2055,17 @@ function Oscar.direct_product(C::GModule...; task::Symbol = :none)
   end
 end
 
-export GModule, gmodule, word, fp_group, confluent_fp_group, induce,
-       action, cohomology_group, extension, is_coboundary, pc_group
+export GModule
```

Comment:
So it seems we are sorting uppercase before lowercase? Then the styleguide should perhaps mention that, too?